.circleci/config.yml

```diff
@@ -174,6 +174,23 @@ jobs:
           name: Publish to npm
           command: yarn release:next --yes
 
+  publish-docs:
```

Comment:
if we intend to continue publishing neo2 docs, we should name this `publish docs-next` or something to signal it.